app/cafe.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        exception_counter = 0
```

Comment:
why do u need this variable ? it seems to me, the work of method will be interupted in case of error